Onboarding/day-three-strategy-doc.md

```diff
@@ -0,0 +1,112 @@
+# Day Three VSP Strategy Meeting
+
+The purpose of this document is to strategize what items we should be discussing and demonstrating on Day Three to be productive. This could include things we want to know about the onboarding team, and things we want the onboarding team to provide during discussion.
+
+**Table of Contents**
+
+- [Day Three VSP Strategy Meeting](#day-three-vsp-strategy-meeting)
+  - [TODO Items to Iterate On](#todo-items-to-iterate-on)
+  - [Before Strategy Meeting](#before-strategy-meeting)
+    - [What does the onboarding team need to do in advance?](#what-does-the-onboarding-team-need-to-do-in-advance)
+    - [What does the VSP team need to do in advance for every team?](#what-does-the-vsp-team-need-to-do-in-advance-for-every-team)
+  - [Agenda](#agenda)
+  - [Strategy Meeting Part I: Discovery](#strategy-meeting-part-i-discovery)
+    - [Goals](#goals)
+    - [Learn About the Product](#learn-about-the-product)
+    - [Break](#break)
+  - [Strategy Meeting Part II: Demo Ticket Creation in Zenhub](#strategy-meeting-part-ii-demo-ticket-creation-in-zenhub)
+  - [Strategy Meeting Part III: Fostering Open Communication](#strategy-meeting-part-iii-fostering-open-communication)
+    - [Break](#break-1)
+  - [Strategy Meeting Part IV: Demo Pull Request Creation in Github](#strategy-meeting-part-iv-demo-pull-request-creation-in-github)
+
+## TODO Items to Iterate On
+
+- ~~Create templates for adding points of contact, project specifics~~
+- ~~Create a meeting notes template for Day Three discovery~~
+- Define logistics of the day. Is this a one-hour meeting, is it all-day? What's the breakdown?
+- Define project folders that are not in the public repo
+
+## Before Strategy Meeting
+
+These are the items that need to be completed before the Day Three meeting(s). Having these items finished will make Day Three go smoothly and help teams learn more in discovery.
+
+### What does the onboarding team need to do in advance?
+
+- Fill out the intake questionnaire
+
+### What does the VSP team need to do in advance for every team?
+
+- Review the intake questionnaire
+- Start a project folder in [sensitive team repo](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive)
+- Copy the sample-product-contacts file into project folder **(TODO: Add URL when PR is approved)**
+- Copy the sample-product-discovery-questions file into project folder **(TODO: Add URL when PR is approved)**
+
+## Agenda
+
+- Getting set up
+- Discovery
+  - Define points of contact
+  - Ask sample questions
+  - Discuss implementation
+- First break
```

Comment:
💯 for turning this into more of a workshop, than a meeting.  I would run with that, and frame it as such, i.e. change the title of things from "meeting" to "workshop", etc.

Reply:
👍